bs/bs.py

```diff
@@ -400,4 +400,40 @@ def remove_codes(self, text : str):
             await ctx.send(embed = self.goodEmbed("Club info successfully edited!"))
         except KeyError:
             await ctx.send(embed = self.badEmbed(f"{key.title()} isn't saved club in this server!"))
-        
+
+    @commands.guild_only()
+    @commands.has_permissions(administrator=True)
+    async def sortroles(self, ctx):
+        for member in ctx.guild.members:
+            if discord.utils.get(member.roles, name='Guest') is not None:
```

Comment:
get the Guest role from guild.roles once before the for loop and then check if it's in member.roles